src/charm.py

```diff
@@ -124,6 +124,11 @@ def on_remote_write_changed(self, _: RelationChangedEvent) -> None:
         self._update_config()
         self._update_status()
 
+    def on_remote_write_broken(self, _: RelationBrokenEvent) -> None:
```

Comment:
This is unused.
If needed, then possibly we'd also need to register relation departed.